recipes/uncommitted/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "uncommitted" %}
+{% set version = "1.6" %}
+{% set md5 = "87a357627ed16a9f02498cea9c61cbb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
```

Comment:
Why skip on Python 3.x? Also why won't this work on Windows? If no reason, can we please drop this?